docs/corsair-commander-guide.md

```diff
@@ -5,7 +5,7 @@ _Driver API and source code available in [`liquidctl.driver.commander_pro`](../l
 ## Initializing the device
 
 _Changed in 1.9.0: the firmware and bootloader versions are not available when
-data is read from [Linux hwmon]._  
+data is read from [Linux hwmon]._
```

Comment:
I prefer to keep the _meaningful_[^1] trailing 2 spaces to avoid forgetting about them later, when new "new in"/"changed in" notes are added.

That said, I know it's quite clunky, and still obscure for most Markdown users, so I'll try to replace all these cases by `<br>`.  (So no need for you to revert this change).

[^1]: As a reminder, the 2 trailing spaces force a line break, without also starting a new paragraph, which is nicer for cases like these "new in"/"changed in" lists.